### PR TITLE
NMS-12927:  Replace OpenBMP code

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12927

### CODE REVIEW

**Comment by cgorantla** [comment]
@Bonrob2  Pleas refer to this commit for docs https://github.com/OpenNMS/opennms/pull/3322/commits/e302dafeca5d290c9a2aaf2595140821827d8d3e

File: opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/bmp.adoc [hunk 1]
```diff
@@ -151,30 +152,34 @@ persisted in OpenNMS Postgres DB.
 |===
 
 Stats and retrieving data from external data for rpki/routeinfo/asinfo are handled only on {opennms-product-name}.
-For this `opennms-telemetry-bmp-stats` feature need to be installed on {opennms-product-name}.
+The `opennms-telemetry-bmp-stats` feature needs to be installed on {opennms-product-name}.
+
+BMP Stats are stored as time series data in Postgres with TimescaleDB extension.
```
Comment (cgorantla) [inline opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/bmp.adoc:157]: @Bonrob2 I missed adding TimescaleDB related info to docs in previous commit. Added here.

**Review by RangerRick — APPROVED** [review]
I can't say I understand BMP in enough depth to be too helpful with the architecture of this, but the code looks OK to me, and the installation bits seem reasonable.

File: opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/bmp.adoc (hunk 1 above)
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/bmp.adoc:157]: LGTM!

**Pull request merged**